ethers-contract/ethers-contract-derive/src/utils.rs

```diff
@@ -6,6 +6,35 @@ use syn::{
     PathArguments, Type,
 };
 
+/// Parses the specified attributes from a `syn::Attribute` iterator.
+macro_rules! parse_attributes {
+    ($attrs:expr, $attr_ident:literal, $meta:ident, $($field:pat, $opt:expr => $block:block)*) => {
```

Comment:
syn 2 has a few breaking changes when it comes to extracting attributes, this seems like a useful helper to have